lib/Inkscape.js

```diff
@@ -93,26 +93,31 @@ class Inkscape extends Stream {
       } else {
         this.inkscapeArgs.push(inkscapeArg);
       }
+    });
 
-      if (setOutputFormat) {
-        if (this.guiMode) {
-          throw new Error(
-            `Cannot use ${setOutputFormat} when --verb=... is in use. Please use --FileSave instead`
-          );
-        }
-        this.inkscapeOutputFilePath = getTemporaryFilePath({
-          suffix: `.${this.outputFormat}`,
-        });
-        setOutputFormat = false;
-      } else if (setInputFormat) {
-        this.inkscapeInputFilePath = getTemporaryFilePath({
-          suffix: `.${this.inputFormat || 'svg'}`,
-        });
-        this.inkscapeArgs.push(this.inkscapeInputFilePath);
-        setInputFormat = false;
+    // Set output format as applicable
+    //
+    if (setOutputFormat || !!outputFormat) {
```

Comment:
Unnecessary `!!` as `if` conditions are already evaluated in a boolean context:

```suggestion
    if (setOutputFormat || outputFormat) {
```

Reply:
For sure. Vim'd too fast on the copy/paste of these guys; I usually will add `!!` on variable assignment to keep the typing consistent, which is why that was there in the first place. Thanks for catching that. (: